src/test/java/org/threeten/extra/chrono/TestInternationalFixedChronology.java

```diff
@@ -0,0 +1,1191 @@
+            { InternationalFixedDate.of(2008, false, true), ChronoField.ALIGNED_DAY_OF_WEEK_IN_MONTH, 0 },
+
+            { InternationalFixedDate.of(2007, false, true), ChronoField.ALIGNED_DAY_OF_WEEK_IN_YEAR, 0 },
+            { InternationalFixedDate.of(2008, true, false), ChronoField.ALIGNED_DAY_OF_WEEK_IN_YEAR, 0 },
+            { InternationalFixedDate.of(2008, false, true), ChronoField.ALIGNED_DAY_OF_WEEK_IN_YEAR, 0 },
+
+            { InternationalFixedDate.of(2007, false, true), ChronoField.ALIGNED_WEEK_OF_MONTH, 0 },
+            { InternationalFixedDate.of(2008, true, false), ChronoField.ALIGNED_WEEK_OF_MONTH, 0 },
+            { InternationalFixedDate.of(2008, false, true), ChronoField.ALIGNED_WEEK_OF_MONTH, 0 },
+
+            { InternationalFixedDate.of(2007, false, true), ChronoField.ALIGNED_WEEK_OF_YEAR, 0 },
+            { InternationalFixedDate.of(2008, true, false), ChronoField.ALIGNED_WEEK_OF_YEAR, 0 },
+            { InternationalFixedDate.of(2008, false, true), ChronoField.ALIGNED_WEEK_OF_YEAR, 0 },
+
+            { InternationalFixedDate.of(2007, false, true), ChronoField.MONTH_OF_YEAR,  0 },
+            { InternationalFixedDate.of(2008, true, false), ChronoField.MONTH_OF_YEAR,  0 },
+            { InternationalFixedDate.of(2008, false, true), ChronoField.MONTH_OF_YEAR,  0 },
+
+            { InternationalFixedDate.of(2007, false, true), ChronoField.PROLEPTIC_MONTH, 2007 * 13 - 1 },
+            { InternationalFixedDate.of(2008, true, false), ChronoField.PROLEPTIC_MONTH, 2008 * 13 - 1 },
+            { InternationalFixedDate.of(2008, false, true), ChronoField.PROLEPTIC_MONTH, 2008 * 13 - 1 },
+
+            { InternationalFixedDate.of(2007, false, true), ChronoField.YEAR, 2007 },
+            { InternationalFixedDate.of(2008, true, false), ChronoField.YEAR, 2008 },
+            { InternationalFixedDate.of(2008, false, true), ChronoField.YEAR, 2008 },
+
+            { InternationalFixedDate.of(2007, false, true), ChronoField.ERA, 1 },
+            { InternationalFixedDate.of(2008, true, false), ChronoField.ERA, 1 },
+            { InternationalFixedDate.of(2008, false, true), ChronoField.ERA, 1 },
+
+            { InternationalFixedDate.of(2007, false, true), ChronoField.DAY_OF_WEEK, 0 },
+            { InternationalFixedDate.of(2008, true, false), ChronoField.DAY_OF_WEEK, 0 },
+            { InternationalFixedDate.of(2008, false, true), ChronoField.DAY_OF_WEEK, 0 },
+        };
+    }
+
+    @Test(dataProvider = "getLong")
+    public void test_data_getLong(
+            final InternationalFixedDate date,
+            final TemporalField field,
+            final long expected) {
+        assertEquals(date.getLong(field), expected);
+    }
+
+    //-----------------------------------------------------------------------
+    // InternationalFixedDate.with
+    //-----------------------------------------------------------------------
+    @DataProvider(name = "with")
+    Object[][] data_with() {
```

Comment:
Need tests dealing with unusual values, like `0` for day-of-week.  See similar tests in Discordian.